ArduCopter/Copter.h

```diff
@@ -806,7 +806,7 @@ class Copter : public AP_Vehicle {
     // mode.cpp
     bool set_mode(Mode::Number mode, ModeReason reason);
     bool set_mode(const uint8_t new_mode, const ModeReason reason) override;
-    uint8_t get_mode() const override { return (uint8_t)control_mode; }
+    uint8_t get_mode() const override { return (uint8_t)flightmode->mode_number(); }
```

Comment:
Not for this PR but I think we can delete get_mode() can't we?  I don't think it's used.